peregrine/defaults.py

```diff
@@ -1,4 +1,5 @@
 # Copyright (C) 2014 Swift Navigation Inc.
```

Comment:
It makes sense to me. The duplicated functionality should be standardized. There are some parameters in initSetting.py, which do not have command line arguments counterparts: ephemMaxAge, acqThreshold etc. I would leave those in initSetting.py and remove the duplicates, which already present in command line parameters.